app/src/views/CountryProfileRiskWatch/index.tsx

```diff
@@ -38,7 +43,10 @@ export function Component() {
     const [
         selectedMonths,
         setSelectedMonths,
-    ] = useInputState<Record<number, boolean> | undefined>({ 0: true });
+    ] = useInputState<(typeof multiMonthSelectDefaultValue) | undefined>({
+        ...multiMonthSelectDefaultValue,
+        [getCurrentMonth()]: true,
+    });
```

Comment:
The default month is not selected. Is there any other requirement?

Reply:
It is selected only after the data is fetched from the server.

NOTE: This change is for a country profile risk watch while the other change is for global risk watch.